package.json

```diff
@@ -127,6 +127,7 @@
       "no-buffer-constructor": "warn",
       "no-console": "off",
       "no-multi-spaces": "error",
+      "no-prototype-builtins": "warn",
```

Comment:
Why is this change here?

Reply:
See comment message - it's a check that's enabled in eslint-6.8 by default and is error-level.